File: app/src/main/java/com/example/expensemanager/model/Finance.java
package com.example.expensemanager.model;

public class Finance {
    String name;
    String type;
    String category;
    String date;
    Double cost;

    public Finance() {}

    public Finance(String name, String type, String category, String date, Double cost) {
        this.name = name;
        this.type = type;
        this.category = category;
        this.date = date;
        this.cost = cost;
    }

    public String getName() {
        return name;
    }

    public String getType() {
        return type;
    }

    public String getCategory() {
        return category;
    }

    public String getDate() {
        return date;
    }

    public Double getCost() {
        return cost;
    }

}

File: app/src/main/java/com/example/expensemanager/model/database/Finance.kt
package com.example.expensemanager.model.database

data class Finance(
    val name: String,
    val typeId: Long,
    val categoryId: Long,
    val date: String,
    val cost: Double
) {
    fun toDbEntity(): DBFinances = DBFinances(
        id = 0,
        name = name,
        typeId = typeId,
        categoryId = categoryId,
        date = date,
        cost = cost
    )
}
